fails to build with freetype2 >= 2.5.1

This problem was reported to Debian as “swfmill: FTBFS: swft/swft_import_ttf.cpp:17:31: fatal error: freetype/tttables.h: No such file or directory”. It’s available at http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=733356

Here’s the patch I submitted to the Debian maintainer. It is for 0.3.2 but the current master branch here has the same line.

``` patch
Description: Fix build failure with freetype 2.5.1
Author: Juhani Numminen <juhaninumminen0@gmail.com>
Bug-Debian: http://bugs.debian.org/733356

--- a/src/swft/swft_import_ttf.cpp
+++ b/src/swft/swft_import_ttf.cpp
@@ -14,7 +14,7 @@
 #include FT_OUTLINE_H
 #include "SWFShapeMaker.h"

-#include <freetype/tttables.h>
+#include FT_TRUETYPE_TABLES_H

 using namespace SWF;

```

--- a/src/swft/swft_import_ttf.cpp
+++ b/src/swft/swft_import_ttf.cpp
@@ -14,7 +14,7 @@
 #include FT_OUTLINE_H
 #include "SWFShapeMaker.h"
 
-#include <freetype/tttables.h>
+#include FT_TRUETYPE_TABLES_H
 
 using namespace SWF;
 

